bazel-mode.el

```diff
@@ -21,6 +21,39 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-mode nil
+  "Major mode for Bazel BUILD files."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-mode-buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'file
+  :group 'bazel-mode
+  :link "https://github.com/bazelbuild/buildtools/tree/master/buildifier")
```

Comment:
This syntax is not correct, see https://www.gnu.org/software/emacs/manual/html_node/elisp/Common-Keywords.html

Reply:
Done.